src/scripts/summary.js

```diff
@@ -410,17 +410,29 @@ class Summary extends H5P.EventDispatcher {
     for (const section of sections) {
       const sectionRow = document.createElement("li");
       sectionRow.classList.add('h5p-interactive-book-summary-overview-section-details');
+      const sectionRowContainer = document.createElement("div");
```

Comment:
Please try not to use double quotes. I know that the ruleset of eslint here does not follow the H5P coding style guide, so your IDE will not complain - and the code styling will eventually need to get fixed anyway - but we can make it right right away :-)

Reply:
It will be taken care of in next commit.